components/Card.js

```diff
@@ -2,22 +2,29 @@ import styled from "styled-components";
 import { useState } from "react";
 import { Button } from "./Button";
 
-export default function Card({ voc }) {
+export default function Card({ voc, handleHit, handleView }) {
   const [showTranslation, setShowTranslation] = useState(false);
 
   return (
     <StyledSection>
       <p style={{ whiteSpace: "pre-wrap" }}>
-        {voc.word}
+        {voc.word} {voc.views} {voc.hits}
         {showTranslation && `\n = \n ${voc.translation}`}
       </p>
       {!showTranslation && (
         <Button
-          onClick={() =>
+          onClick={() => {
+            handleHit(voc.id);
+            handleView(voc.id);
             setShowTranslation(
               (previousShowTranslation) => !previousShowTranslation
-            )
-          }
+            );
+          }}
+          // onClick={() =>
+          //   setShowTranslation(
+          //     (previousShowTranslation) => !previousShowTranslation
+          //   )
+          //}
```

Comment:
This commented out area can probably be removed.

Reply:
Of cause! I am still wondering, if that was a good practice, to modify the card-component since this was not part of the userstory but only a kind of debugging to see if it works.

Reply:
Well, imagine if we still had Quality Assurance. How else would they test this US?